gigpress.php

```diff
@@ -117,8 +117,12 @@ function gigpress_admin_head()	{
 
 
 function gigpress_admin_footer() {
-
-	return (__("You're using", "gigpress").' <a href="http://gigpress.com">GigPress '.GIGPRESS_VERSION.'</a>. '. __("Like it?", "gigpress") . ' <a href="http://gigpress.com/donate">' . __("Make a donation", "gigpress") . '</a>.');
+	return sprintf(
+		__( 'You&#146;re using %1$sGigPress %2$s%3$s.', 'gigpress' ),
```

Comment:
It's normally better to split it into two `_x` string and a simple string for the link HTML, also with `target="_blank"`.

Preventing translators from breaking the link structure.
```
$link = sprintf( '<a href="http://gigpress.com" target="_blank">%1$s</a>', esc_html_x( 'GigPress', 'Admin Footer link label', 'gigpress' );
return sprintf( _x( 'You&#146;re using %1$s', 'Admin Footer', 'gigpress' ), $link );
```